src/dash-table/components/ControlledTable/index.tsx

```diff
@@ -817,7 +818,7 @@ export default class ControlledTable extends PureComponent<ControlledTableProps>
                 <div
                     ref='table'
                     className={innerClasses.join(' ')}
-                    style={tableStyle}
+                    style={R.omit(BORDER_PROPERTIES_AND_FRAGMENTS, tableStyle)}
```

Comment:
Omit the border stuff on the inner container and only apply to the outer container - otherwise the border gets doubled.